src/vellum/workflows/events/workflow.py

```diff
@@ -169,6 +172,21 @@ def outputs(self) -> OutputsType:
     def final_state(self) -> Optional[StateType]:
         return self.body.final_state
 
+    @field_serializer("body")
+    def serialize_body(
+        self, body: WorkflowExecutionFulfilledBody[OutputsType, StateType], info: SerializationInfo
+    ) -> WorkflowExecutionFulfilledBody[OutputsType, StateType]:
+        context = info.context if info and hasattr(info, "context") else {}
+        if context and "event_enricher" in context and callable(context["event_enricher"]):
```

Comment:
we'll need to update the method in the `vellum_ee` module to enable adding server metadata for fulfilled events. we likely should add a test for this